src/helpers/validate.helper.js

```diff
@@ -0,0 +1,23 @@
+import { check } from 'express-validator';
+
+/**
+ * A class that contains functions for all validations
+ */
+class Validate {
+  /**
+ * Adds two numbers together.
```

Comment:
### Critical

- Is that an appropriate description?

Reply:
I apologise for this mistake.  I forgot to edit this comment I got from jsDoc documentation website. it wont happen again.